src/components/configuration/ActionLinkConfig.vue

```diff
@@ -37,18 +34,49 @@
 
         <v-text-field
           v-model="dialog.minInterval"
-          label="Minimum interval between calls (ms)"
+          label="Minimum interval between variable-based calls (ms)"
           type="number"
           min="0"
           variant="outlined"
           density="compact"
           class="mt-4"
         />
 
+        <div class="mt-4">
+          <p class="text-subtitle-1 font-weight-bold mb-2">Run time-based:</p>
+          <v-radio-group v-model="dialog.autoRunType" class="mt-2">
+            <v-radio value="none" label="Do not run automatically"></v-radio>
+            <v-radio value="once" label="Run automatically once"></v-radio>
+            <v-radio value="interval" label="Run automatically frequently"></v-radio>
+          </v-radio-group>
+
+          <v-text-field
+            v-if="dialog.autoRunType === 'once'"
+            v-model="startupDelaySeconds"
+            label="Delay after startup (s)"
+            type="number"
+            min="0"
+            variant="outlined"
+            density="compact"
+            class="mt-2"
+          />
+
+          <v-text-field
+            v-if="dialog.autoRunType === 'interval'"
+            v-model="runFrequencySeconds"
+            label="Run every (s)"
+            type="number"
+            min="0"
+            variant="outlined"
+            density="compact"
+            class="mt-2"
+          />
```

Comment:
```suggestion
          <p class="text-subtitle-1 font-weight-bold mb-2">Time-based:</p>
          <v-radio-group v-model="dialog.autoRunType" class="mt-2">
            <v-radio value="once" label="On startup"></v-radio>
            <v-radio value="interval" label="Repeatedly"></v-radio>
            <v-radio value="none" label="No time trigger"></v-radio>
          </v-radio-group>

          <v-text-field
            v-if="dialog.autoRunType !== 'none'"
            v-model="runFrequencySeconds"
            :label="dialog.autoRunType === 'interval' ? 'Run every' :  'Run after'"
            suffix="s"
            type="number"
            min="0"
            variant="outlined"
            density="compact"
            class="mt-2"
          />
```
Given the similarities between the interval and once options, it seems cleaner to combine them and just make the label dynamic. Will require updating the code to use a single variable.